fixed education page alignment. Used css grids to show awards and prizes

Ayush0730 opened the pull request:
### 1. Result image is attached. I have removed the carousel and added grid system
![image](https://user-images.githubusercontent.com/68139755/105154243-5a35d500-5b2f-11eb-8ba1-6f599963b133.png)

### 2. Education details were overflowing over borders in small and medium sizes, fixed that
**Before**
![image](https://user-images.githubusercontent.com/68139755/105156384-c6b1d380-5b31-11eb-8fe0-1c0fb6505c77.png)

**After** 
![image](https://user-images.githubusercontent.com/68139755/105156450-dcbf9400-5b31-11eb-8743-d24e321b9986.png)

smaranjitghose commented:
> ### 1. Result image is attached. I have removed the carousel and added grid system
> ![image](https://user-images.githubusercontent.com/68139755/105154243-5a35d500-5b2f-11eb-8ba1-6f599963b133.png)
> 
> ### 2. Education details were overflowing over borders in small and medium sizes, fixed that
> **Before**
> ![image](https://user-images.githubusercontent.com/68139755/105156384-c6b1d380-5b31-11eb-8fe0-1c0fb6505c77.png)
> 
> **After**
> ![image](https://user-images.githubusercontent.com/68139755/105156450-dcbf9400-5b31-11eb-8743-d24e321b9986.png)

I meant remove the entire Awards Section. Rest is fine

Ayush0730 commented:
> > ### 1. Result image is attached. I have removed the carousel and added grid system
> > ![image](https://user-images.githubusercontent.com/68139755/105154243-5a35d500-5b2f-11eb-8ba1-6f599963b133.png)
> > ### 2. Education details were overflowing over borders in small and medium sizes, fixed that
> > **Before**
> > ![image](https://user-images.githubusercontent.com/68139755/105156384-c6b1d380-5b31-11eb-8fe0-1c0fb6505c77.png)
> > **After**
> > ![image](https://user-images.githubusercontent.com/68139755/105156450-dcbf9400-5b31-11eb-8743-d24e321b9986.png)
> 
> I meant remove the entire Awards Section. Rest is fine

> but I think we should keep the awards section, if someone wants to show their awards

smaranjitghose commented:
> > > ### 1. Result image is attached. I have removed the carousel and added grid system
> > > ![image](https://user-images.githubusercontent.com/68139755/105154243-5a35d500-5b2f-11eb-8ba1-6f599963b133.png)
> > > ### 2. Education details were overflowing over borders in small and medium sizes, fixed that
> > > **Before**
> > > ![image](https://user-images.githubusercontent.com/68139755/105156384-c6b1d380-5b31-11eb-8fe0-1c0fb6505c77.png)
> > > **After**
> > > ![image](https://user-images.githubusercontent.com/68139755/105156450-dcbf9400-5b31-11eb-8743-d24e321b9986.png)
> > 
> > 
> > I meant remove the entire Awards Section. Rest is fine
> 
> > but I think we should keep the awards section, if someone wants to show their awards

We will design a custom carousel for it
